check.js

```diff
@@ -0,0 +1,36 @@
+import fs from 'fs';
```

Comment:
This is great. probably better would be to make this a unit test

This way, we can run it as a Github Action. In the future can be evolved to be a merge criteria.

I would foresee we will end up creating a git hook that checks moved MD files and ensures everything there is in the CSV before we commit. If we have the test that checks that everything in the CSV is in the redirect, then we make sure is impossible to make 404s if we move folders around